packages/data-store/src/entities/statusList/StatusListEntities.ts

```diff
@@ -110,6 +110,6 @@ export class StatusList2021Entity extends StatusListEntity {
 export class OAuthStatusListEntity extends StatusListEntity {
   @Column({ type: 'integer', name: 'bitsPerStatus', nullable: false })
   bitsPerStatus!: number
-  @Column({ type: 'datetime', name: 'expiresAt', nullable: true })
+  @Column({ type: process.env.DB_TYPE === 'postgres' ? 'timestamp' : 'datetime', name: 'expiresAt', nullable: true })
```

Comment:
We have a function for that that we use elsewhere for dates as well

Reply:
typeOrmDateTime()